openhands_cli/argparsers/main_parser.py

```diff
@@ -117,4 +120,30 @@ def create_main_parser() -> argparse.ArgumentParser:
         "acp", help="Start OpenHands as an Agent Client Protocol (ACP) agent"
     )
 
+    # Add login subcommand
+    login_parser = subparsers.add_parser(
+        "login", help="Authenticate with OpenHands Cloud using OAuth 2.0 Device Flow"
+    )
+    default_cloud_url = os.getenv("OPENHANDS_CLOUD_URL", "https://app.all-hands.dev")
+    login_parser.add_argument(
+        "--server-url",
+        type=str,
+        default=default_cloud_url,
+        help=(
+            f"OpenHands server URL (default: {default_cloud_url}, "
+            "configurable via OPENHANDS_CLOUD_URL env var)"
+        ),
+    )
+
+    # Add logout subcommand
+    logout_parser = subparsers.add_parser("logout", help="Log out from OpenHands Cloud")
+    logout_parser.add_argument(
+        "--server-url",
+        type=str,
+        help=(
+            "OpenHands server URL to log out from "
+            "(if not specified, logs out from all servers)"
+        ),
+    )
+
```

Comment:
@openhands move this to a file of its own

then attach in in this file (see the pattern for the server parser)

also fix the server parser since its defined, call the helper function here instead